dali/kernels/test/kernel_test_utils.h

```diff
@@ -29,21 +29,102 @@ struct SimpleKernelTestBase {
 
   template <int i>
   using Input =  std::remove_reference_t<
-      std::tuple_element_t<i, kernels::kernel_inputs<Kernel>>>;
+          std::tuple_element_t<i, kernels::kernel_inputs < Kernel>>>;
```

Comment:
Is it supposed to be a comparison expression between kernels::kernel_inputs `less than` Kernel?